### ISSUE TITLE
Create OnlyOffice Application Template

### ISSUE THREAD

**Issue by nicholas-itpedia** [clouder-community/clouder#149]
Create OnlyOffice Application Template.
http://www.onlyoffice.com/features.aspx

### Reference:
- http://www.onlyoffice.com/blog/2015/02/how-to-install-onlyoffice-in-just-a-few-minutes-using-docker-service/
- https://github.com/ONLYOFFICE

**Comment by YannickB** [comment]
Yeah we discussed about it, I didn't know and it looks really well to provide online office features, we definitely need it.

**Comment by lasley** [comment]
Yeah I like this. My solution was going to be OwnCloud, but I think this looks way better 👍
